scripts/autoscaling/azure/nodeup.py

```diff
@@ -358,33 +360,46 @@ def get_os_profile(instance_name, ssh_pub_key, user, user_data_script, computer_
     return profile
 
 
-def get_storage_profile(disk, image, instance_type):
+def get_data_disk(size, type, lun, disk_name=None):
+    disk = {
+                "diskSizeGB": size,
+                "lun": lun,
+                "createOption": "Empty",
+                "managedDisk": {
+                    "storageAccountType": type
```

Comment:
maybe it would be better to call this parameter like `disk_type` (and the others if it exists) to avoid using reserved python words as parameter name

Reply:
Yes, fixed.